general-concepts/dependencies/text.xml

```diff
@@ -218,54 +215,68 @@ to the package manager.
 </p>
 
 <p>
-The following description applies to all EAPIs starting with EAPI 2.
+There are two kinds of blockers: <e>weak blockers</e> and <e>strong
+blockers</e>.
 </p>
 
 <p>
-There are two kinds of blockers: soft blockers and hard blockers.
+A weak blocker is defined using the following syntax:
 </p>
 
-<p>A soft blocker is defined using the following syntax:</p>
-
 <codesample lang="ebuild">
 RDEPEND="!app-misc/foo"
 </codesample>
 
 <p>
 The package manager will try to resolve this conflict automatically.
-The package blocked by a soft blocker can be uninstalled <e>after</e>
+The package blocked by a weak blocker can be uninstalled <e>after</e>
 installing the package blocking it. However, it exempts the common
-files from file collision checks. Soft blockers are usually used
+files from file collision checks. Weak blockers are usually used
 to solve file collisions between packages and are meaningful only
 in <c>RDEPEND</c>.
 </p>
 
+<p>
+More specifically, installation of the newer package may overwrite any
+colliding files that belong to the older package which is explicitly blocked.
```

Comment:
```suggestion
colliding files that belong to the older package that is explicitly blocked.
```

Reply:
Updated. (The "which" was in the text that I've copied, though.)